cli/cmd/namespace.go

```diff
@@ -12,6 +12,7 @@ type envSettings struct {
 
 // injected monitor annotation
 const marblerunAnnotation = "marblerun/monitor"
+const injectionAnnotation = "marblerun/injectsgx"
```

Comment:
Oh, wait, misunderstanding on my side. This annotation is used to mark where we want to inject sgx-device-plugin-specific stuff, right?

Reply:
Yes

Reply:
Maybe we should renamed the "marblerun/monitor" label to something else, since that label is used to determine whether or not to inject anything at all.
Having two labels: "marblerun/inject" and "marblerun/inject-sgx" might be a better approach
The first to determine if we should inject at all and the second if we should additionally inject the sgx tolerations

Reply:
sounds good to me